doc/content/verification_and_validation/ver-1e.md

```diff
@@ -45,23 +45,32 @@ $k$ = $\sqrt{\frac{D_{PyC}}{D_{SiC}}}$
 
 and $\lambda_n$ are the roots of
 
-$tan(\lambda a) + k \; tan(k l \lambda) = 0$
+$\frac{1}{tan(\lambda a)} + \frac{1}{k \; tan(k l \lambda)} = 0$
 
+NOTE: The above expression for the roots of $\lambda$ is obtained from [!cite](longhurst1992verification). The equation presented in [!cite](ambrosek2008verification) has a typographical error and gives incorrect results.
 
 ## Results
 
-[ver-1e_comparison_dist] shows the comparison of the TMAP8 calculation and the analytical solution for concentration after steady-state is reached.
+[ver-1e_comparison_dist] shows the comparison of the TMAP8 calculation and the analytical solution for concentration after steady-state is reached. The plots show the TMAP8 and analytical solution comparisons for both the TMAP4 case ($l$ = 63 $\mu m$) and the TMAP7 case ($l$ = 66 $\mu m$). 
 
 !media figures/ver-1e_comparison_dist.png
     style=width:50%;margin-bottom:2%
     id=ver-1e_comparison_dist
-    caption=Comparison of TMAP8 calculation with the analytical solution
+    caption=Comparison of TMAP8 calculation with the analytical solution. Bold text next to the plot curves shows the error between the TMAP8 prediction and analytical solution for the TMAP4 and TMAP7 verification cases.
```

Comment:
```suggestion
    caption=Comparison of TMAP8 calculation with the analytical solution. Bold text next to the plot curves shows the root mean square percentage error (RMSPE) between the TMAP8 prediction and analytical solution for the TMAP4 and TMAP7 verification cases.
```
$R$ is often reserved for RMSE, maybe use RMSPE (root mean square percentage error) in the figures.